webpack.config.js

```diff
@@ -1,65 +1,75 @@
-const path = require('path');
-const webpack = require('webpack');
-const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
-const ExtractTextPlugin = require("extract-text-webpack-plugin");
+/**
+ * @format
+ */
 
-const extractSass = new ExtractTextPlugin({
-  filename: "../css/[name].css"
-});
-
-const uglifyJs = new UglifyJsPlugin({
-});
+const path = require("path");
+const webpack = require("webpack");
+const TerserJSPlugin = require("terser-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
 
 module.exports = {
+  optimization: {
+    minimizer: [new TerserJSPlugin({}), new OptimizeCSSAssetsPlugin({})],
+    splitChunks: {
+      cacheGroups: {
+        styles: {
+          name: "main",
+          test: /\.css$/,
+          chunks: "all",
+          enforce: true
+        }
+      }
+    }
+  },
   entry: "./webpack/entry.js",
   output: {
     path: path.resolve(__dirname, "js/"),
     filename: "bundle.js"
   },
   module: {
-    loaders:[
-      {
-        test: /\.jsx?$/,
-        exclude: /(node_modules)/,
-        loader: "babel-loader",
-        query: {
-          presets: ["react", "es2015"]
-        }
-      }
-    ],
     rules: [
       {
-        test: /\.(ttf|eot|svg|gif)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-        exclude: /node_modules/,
-        use: {
-          loader: "file-loader"
-        }
-      },
-      {
-        test: /\.scss$/,
-        use: extractSass.extract({
-          use: [
-            {
-              loader: "css-loader",
-              options: {
-                minimize: true,
+        test: /\.scss?/,
+        use: [
+          {
```

Comment:
sass-loader actually required postcss-loader with autoprefixer, so I don't know what thats about but I would rather include it.